src/CMakeLists.txt

```diff
@@ -153,10 +153,11 @@ function(add_runtime_modules TARGET_RUNTIME_FILES TARGET_RUNTIME_DIR)
       set(BC "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.build/${NATIVE_INT_DIR}initmod.${i}_${j}.bc")
       set(INITMOD_D "${INITMOD_PREFIX}${i}_${j}_debug.cpp")
       set(INITMOD "${INITMOD_PREFIX}${i}_${j}.cpp")
+      set(INCLUDE_DIRS -I${TARGET_RUNTIME_DIR})
```

Comment:
Added quoting at the def, but didn't add it at the usage since it doesn't work with a list (it emits semi-colon separated list). Also fixed other places with escaped space.